### DISCUSSION TITLE
2024/02/28/Riemann-Integral/

### DISCUSSION THREAD

**Discussion by giscus** [AbyssSkb/abyssskb.github.io#1] [Announcements]
# 2024/02/28/Riemann-Integral/

在实分析中，由黎曼创立的黎曼积分（英语：Riemann integral）首次对函数在给定区间上的积分给出了一个精确定义。黎曼积分在技术上的某些不足之处可由后来的黎曼－斯蒂尔杰斯积分和勒贝格积分得到修补。

https://blog.myabyss.top:8443/2024/02/28/Riemann-Integral/

**Comment by AbyssSkb** [comment]
hello
